src/api/app/controllers/webui/package_controller.rb

```diff
@@ -663,6 +663,16 @@ def remove
     end
   end
 
+  def trigger_services
+    begin
+      Suse::Backend.post "/source/#{@project}/#{@package}?cmd=runservice&user=#{User.current}"
```

Comment:
you need to URI.escape those - devel:libraries:c++ will otherwise not be able to use this feature